/seppia/aruco_detector:
  ros__parameters:
    aruco_side: 0.15
    autostart: true
    base_frame: seppia/base_footprint
    best_effort_sub_qos: true
    centering_width: 150
    excluded_ids:
      - 0
      - 1023
    image_sub_depth: 3
    input_topic: /seppia/zed2i_left_camera/image_raw
    output_topic: /seppia/aruco_detector/targets
    stream_topic: /gui_1
    transport: compressed
    worker_cpu: 4
